.travis.yml

```diff
@@ -0,0 +1,19 @@
+sudo: false
+language: python
+
+python:
+    - "2.7"
+    - "3.4"
+
+env:
+    - TOXENV=django19
```

Comment:
Does this only support Django 1.9? I think it'd be a good idea to make sure this still supports 1.8